src/libraries/System.Threading.Channels/src/System/Threading/Channels/Channel.cs

```diff
@@ -49,7 +49,22 @@ public static Channel<T> CreateBounded<T>(BoundedChannelOptions options)
                 throw new ArgumentNullException(nameof(options));
             }
 
```

Comment:
Delete the above argument validation.  It's now handled by the method we delegate to.

Reply:
Done